### DISCUSSION TITLE
Filter ICS based off parameters E.g Busy/Free

### DISCUSSION THREAD

**Discussion by jumpingbrowntrout** [bitfireAT/icsx5#456] [General]
Hello,

Raising this as an idea that I hope you would consider for a future an enhancement.

My work outlook calendar has meetings from start until end of the day, and often 3 or for other lapped. I need to be aware of these meetings incase I need to join and as such cannot decline.

I mark any reorruring meeting that I am optional for as free instead of busy. 

When some one cancels an outlook event it renames the titled to "Cancelled:..." And changes the availability to free.

I sync this calendar to my phone so that I can see exactly what meetings must attend. But presentually it shows absolutely everything, including everything that's cancelled.

Would you consider the possibility to add some basic sync filters for an ICS calendar so we could filter out things that aren't important?

Cheers :)

**Comment by sunkup** [comment]
Hmm, maybe. I think ideally this should be handled by the calendar app. The app could apply/remove the filter in order to hide/show the events and ICSx5 would not need to do new syncs.
